checkov/terraform/checks/graph_checks/aws/VPCHasOneOfWantedFlowLogs.yaml

```diff
@@ -0,0 +1,34 @@
+metadata:
```

Comment:
@NoaAzoulay This is a test check, it should not go into the default check directory but rather into the test directories